Kernel/Net/Intel/E1000ENetworkAdapter.cpp

```diff
@@ -235,19 +241,18 @@ UNMAP_AFTER_INIT E1000ENetworkAdapter::E1000ENetworkAdapter(StringView interface
     NonnullOwnPtr<Memory::Region> tx_buffer_region, NonnullOwnPtr<Memory::Region> rx_descriptors_region,
     NonnullOwnPtr<Memory::Region> tx_descriptors_region)
     : E1000NetworkAdapter(interface_name, device_identifier, irq, move(registers_io_window),
-        move(rx_buffer_region),
-        move(tx_buffer_region),
-        move(rx_descriptors_region),
-        move(tx_descriptors_region))
+          move(rx_buffer_region),
+          move(tx_buffer_region),
+          move(rx_descriptors_region),
+          move(tx_descriptors_region))
 {
 }
 
 UNMAP_AFTER_INIT E1000ENetworkAdapter::~E1000ENetworkAdapter() = default;
 
 UNMAP_AFTER_INIT void E1000ENetworkAdapter::detect_eeprom()
 {
-    // FIXME: Try to find a way to detect if EEPROM exists instead of assuming it is
-    m_has_eeprom = true;
+    m_has_eeprom = in32(REG_EECD) & EECD_PRES;
```

Comment:
It would be nice to have more details on that. Is there a particular section in the specification?